docs/reference_guide.md

```diff
@@ -1359,7 +1359,7 @@ usdt:library_path:probe_name
 usdt:library_path:[probe_namespace]:probe_name
 ```
 
-Where `probe_namespace` is optional if `probe_name` is unique within the binary.
+Where `probe_namespace` is optional if `probe_name` is unique within the binary. You can also target all shared libraries and the main executable in a process's address space if you use the `*` wildcard in place of the `binary_path`/`library_path` AND pass in a pid command line argument e.g. `bpftrace -e 'usdt:*:loop { printf("hi\n"); }' -p 1234`.
```

Comment:
This applies to uprobes, too, doesn't it? Shouldn't it be placed elsewhere, then?

Reply:
Good point. How about in the probes intro section or perhaps I should add a new "-p" item in the Usage section that points this out.

Reply:
Either *Probes* intro or a new subsection of *Probes* seems the best place to me. I'm also wondering if we should reflect this in manpage, too (we should really put some work to unify these two).

Reply:
Actually, this method of targeting doesn't work for uprobes/uretprobes. Anyway, going to abandon in favor of [this PR](https://github.com/iovisor/bpftrace/pull/2734)